modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm

```diff
@@ -157,18 +156,14 @@
 	var/brute_damage = patient.getBruteLoss()
 	var/burn_damage = patient.getFireLoss()
 
-	if(obj_flags & EMAGGED)
-		patient.monkeyize()
```

Comment:
Tested locally: does not ghost people

Reply:
Seen it ghost people on live

Reply:
> Tested locally: does not ghost people

it ghosted me when I tested it in the tdome soo....